### PR TITLE
Support Pac in ephemeral cluster to  work with QE SprayProxy

### PR DESCRIPTION
**Pull request by xinredhat**
**Background:**
Please see the doc [RHTAP Quality Testing Workflow](https://docs.google.com/document/d/13M5WjmIXwVsExNVK_xk4YINFqohVvZw4gwLAZTPdq-k/edit)

**Changes:**
When deploying RHTAP, Passing PAC_GITHUB_APP_WEBHOOK_SECRET as parameter to configure secret pipelines-as-code-secret. This allows Pac in the ephemeral cluster to handle requests sent from the GitHub App

### CODE REVIEW

File: hack/build/setup-pac-integration.sh [hunk 1]
```diff
@@ -64,7 +64,15 @@ setup-pac-app() (
 )
 
 if [ -n "${PAC_GITHUB_APP_ID}" ] && [ -n "${PAC_GITHUB_APP_PRIVATE_KEY}" ]; then
-        WEBHOOK_SECRET=$(setup-pac-app)
+        # for dubugging purposes
+        echo "PAC_GITHUB_APP_ID: $PAC_GITHUB_APP_ID"
+        echo "PAC_GITHUB_APP_PRIVATE_KEY: $PAC_GITHUB_APP_PRIVATE_KEY"
+        echo "PAC_GITHUB_APP_WEBHOOK_SECRET: $PAC_GITHUB_APP_WEBHOOK_SECRET"
```
Comment (psturc) [inline hack/build/setup-pac-integration.sh:70]: I guess these lines should be removed

Comment (xinredhat) [reply hack/build/setup-pac-integration.sh]: fixed, thanks for the reminder

**Review by ramessesii2 — APPROVED** [review]
/lgtm
/approve

**Review by prietyc123 — APPROVED** [review]

**Pull request merged**